setup.py

```diff
@@ -22,9 +25,11 @@ def getVersionFromFile():
       url='https://github.com/ctlearn-project/ctlearn',
       license='BSD-3-Clause',
       packages=['ctlearn'],
+      install_requires=getRequirements(),
```

Comment:
You should not duplicate information between `pyproject.toml` and setup.py.

This can lead to confusion and errors.


After you introduced `pyproject.toml` with all the configurations you provided, you should just remove the `setup.py`.